pkg/registry/clusterrole/policybased/storage.go

```diff
@@ -19,12 +19,16 @@ package policybased
 
 import (
 	"k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/api/errors"
 	"k8s.io/kubernetes/pkg/api/rest"
+	"k8s.io/kubernetes/pkg/api/unversioned"
 	"k8s.io/kubernetes/pkg/apis/rbac"
 	"k8s.io/kubernetes/pkg/apis/rbac/validation"
 	"k8s.io/kubernetes/pkg/runtime"
 )
 
+var groupResource = unversioned.GroupResource{Group: "rbac.authorization.k8s.io", Resource: "clusterroles"}
```

Comment:
fyi, all the other api groups have a `func Resource(resource string) unversioned.GroupResource` that builds this so the group name doesn't have to be hard coded in lots of places